ext/Client/Presets/Vanilla/MP_007/Night.lua

```diff
@@ -77,7 +77,7 @@ return [[
     "OutdoorLight":{
        	"Enable":"true",
        	"SunRotationX":"0.0",
-		"SunRotationY":"180.0",
+		"SunRotationY": "180.0",
```

Comment:
what happened to all of those? :D

Reply:
Already cleaned that up in the last commit